packages/cloudflare/src/api/cloudflare-context.ts

```diff
@@ -166,12 +168,13 @@ async function getCloudflareContextAsync<
  * with the open-next Cloudflare adapter
  *
  * Note: this function should only be called inside the Next.js config file, and although async it doesn't need to be `await`ed
+ * @param options options how the function should operate and if/where to persist the platform data
  */
-export async function initOpenNextCloudflareForDev() {
+export async function initOpenNextCloudflareForDev(options?: Omit<GetPlatformProxyOptions, "environment">) {
```

Comment:
Why omitting `environment`?

Reply:
Currently in `getCloudflareContextFromWrangler` the environment config is not taken into account because that function already handles it. So I wanted to reflect that in the type so people will not get confused here. If you think the environment should also be configurable I will change it.

```ts
  // getCloudflareContextFromWrangler()
  const { env, cf, ctx } = await getPlatformProxy({
    // This allows the selection of a wrangler environment while running in next dev mode
    ...options,
    environment: process.env.NEXT_DEV_WRANGLER_ENV,
  });
```

Reply:
Ah I see 🤔 

I think it might still be good to allow the user to provide an environment for the platform proxy though 🤔 

Could we remove this limitation? and instead have some code such as:
```ts
const environment = options.environment ?? process.env.NEXT_DEV_WRANGLER_ENV;

if (options.environment && process.env.NEXT_DEV_WRANGLER_ENV) {
    // console.warn saying that NEXT_DEV_WRANGLER_ENV is set but it's being overridden by the environment option
}

// getCloudflareContextFromWrangler()
const { env, cf, ctx } = await getPlatformProxy({
  ...options,
  environment,
 });
```
?

In this way `NEXT_DEV_WRANGLER` still works but developers do have the opportunity to use the `environment` option for `initOpenNextCloudflareForDev`

what do you think? 🙂

Reply:
I agree. I've pushed the change